var/spack/repos/builtin/packages/touchdetector/package.py

```diff
@@ -14,6 +14,7 @@ class Touchdetector(CMakePackage):
     git      = "git@bbpgitlab.epfl.ch:hpc/touchdetector.git"
 
     version('develop', submodules=True)
+    version('5.6.0.remote', branch='remote-filtering', submodules=True)
```

Comment:
not as per `semantic versioning` scheme :) 

Is this temporary for testing or for the deployment? If for deployment, commit id would be better than branch.

Reply:
ah damn. yeah, testing :)

Reply:
```suggestion
```